app/src/main/java/ch/sdp/vibester/activity/GamescreenActivity.kt

```diff
@@ -10,11 +10,35 @@ import android.widget.LinearLayout
 import android.widget.TableRow
 import android.widget.TextView
 import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.children
+import ch.sdp.vibester.BuzzerScoreUpdater
 import ch.sdp.vibester.R
 
 
 class GamescreenActivity: AppCompatActivity() {
 
+    private val MAX_N_PLAYERS = 4
+    private val NO_BUZZER_PRESSED = -1
+    private val rowsIdArray = arrayOf(R.id.row_0, R.id.row_1, R.id.row_2, R.id.row_3)
+    private val buzIds = arrayOf(R.id.buzzer_0, R.id.buzzer_1, R.id.buzzer_2, R.id.buzzer_3)

```

Comment:
Would not really make sense to refactor, this is just declaration and initialisation of 2 arrays with values from the xml.

Reply:
Agreed. Worst case, if these buzzers and rows are the only children of the layout, can iterate over them and use getChild to set their values. Leaving the comment here in case we need to do so in the future.